model/src/test/kotlin/ScanResultContainerTest.kt

```diff
@@ -67,7 +67,7 @@ class ScanResultContainerTest : WordSpec() {
             LicenseFinding(
                 "license-1.1",
                 TextLocation("path 1.1", 1, 1)
-            ),
+            ).apply { this.source = "source 1" },
```

Comment:
Please remove the explicit `this.`. Same below.

Reply:
done (removed by other request)

Reply:
done (removed due to other question)